### PR TITLE
add `constant.ts` version to `version_check.py`

### PR DESCRIPTION
**Pull request by AHReccese**
`version_check.py` script updated [add website's footer version]

### CODE REVIEW

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]
@AHReccese 

Please update `RELEASE.md`(item 2):
    1. `SECURITY.md`
    2. `otherfiles/version_check.py`
    3. `src/App/Config/constants.ts`
    
 Add `package.json` to `version_check.py` script

**Review by F-Shahali — APPROVED** [review]

**Comment by AHReccese** [comment]
> @AHReccese
> 
> Please update `RELEASE.md`(item 2): 1. `SECURITY.md` 2. `otherfiles/version_check.py` 3. `src/App/Config/constants.ts`
> 
> Add `package.json` to `version_check.py` script

Sure thing, I applied them all.

File: otherfiles/RELEASE.md [hunk 1]
```diff
@@ -4,8 +4,10 @@
 
 1. Create the `release` branch under `dev`
 2. Update all version tags
-    1. `package.json`
-    2. `README.md`
+    1. `SECURITY.md`
+    2. `package.json`
+    2. `otherfiles/version_check.py`
+    3. `src/App/Config/constants.ts`
 3. Update `CHANGELOG.md`
```
Comment (sepandhaghighi) [inline otherfiles/RELEASE.md:11]: Add something like this:

4. Update `.github/ISSUE_TEMPLATE/bug_report.yml`
   1. Add new version tag to `MyButton version` dropbox options

Comment (sepandhaghighi) [reply otherfiles/RELEASE.md:11]: Take a look at https://github.com/openscilab/pymilo/blob/main/otherfiles/RELEASE.md

File: otherfiles/RELEASE.md [hunk 2]
```diff
@@ -4,8 +4,10 @@
```
Comment (sepandhaghighi) [inline otherfiles/RELEASE.md:4]: `**Last Update: 2024-07-06**`

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]
@AHReccese Take a look at my comments 🔥

File: otherfiles/RELEASE.md (hunk 2 above)
Comment (AHReccese) [reply otherfiles/RELEASE.md:4]: applied

File: otherfiles/RELEASE.md (hunk 1 above)
Comment (AHReccese) [reply otherfiles/RELEASE.md:11]: applied

File: otherfiles/RELEASE.md [hunk 3]
```diff
@@ -1,16 +1,20 @@
...
-#### Last Update: 2024-06-15
+#### Last Update: 2024-07-14
 
 1. Create the `release` branch under `dev`
 2. Update all version tags
-    1. `package.json`
-    2. `README.md`
+    1. `SECURITY.md`
+    2. `package.json`
+    2. `otherfiles/version_check.py`
```
Comment (sepandhaghighi) [inline otherfiles/RELEASE.md:9]: Duplicate numbers.
1.
2.
3.
4.

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]

**Review by sepandhaghighi — APPROVED** [review]
🔥

**Pull request merged**